core/src/main/java/com/linecorp/armeria/server/Route.java

```diff
@@ -179,4 +179,10 @@ default RoutingResult apply(RoutingContext routingCtx) {
      * Returns the {@link Set} of {@link MediaType}s that this {@link Route} produces.
      */
     Set<MediaType> produces();
+
+    /**
+     * Returns whether this {@link Route} is a fallback, which is matched only when no configured {@link Route}s
+     * were matched.
```

Comment:
was matched

Reply:
Thanks! I removed `s` next to `{@link Route}` as well.